src/components/StructuredContent/StructuredContent.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import { Box, useMultiStyleConfig } from "@chakra-ui/react";
+
+import Heading from "../Heading/Heading";
+import { HeadingLevels } from "../Heading/HeadingTypes";
+import Image, { ImageProps } from "../Image/Image";
+import { ImageRatios, ImageSizes } from "../Image/ImageTypes";
+import generateUUID from "../../helpers/generateUUID";
+
+export enum StructuredContentImagePosition {
+  Left = "left",
+  Right = "right",
+  Center = "center",
+}
+
+export interface StructuredContentProps {
+  calloutContent?: string | JSX.Element;
```

Comment:
I am reconsidering this prop type assignment.  I think we can drop `JSX.Element` and just use `string`.  I would like the formatting of the `Callout` heading to always be consitent.